parameter_tools/src/test/java/JSONSyncTests.java

```diff
@@ -45,4 +44,13 @@ public void JsonSyncSetFileTest() {
         Assertions.assertEquals(16.4, instance.motorData.maxVoltage);
         Assertions.assertEquals(0.0, instance.motorData.currentVoltage);
     }
+
+    @Test
+    public void JsonSyncSaveFileTest() {
+        ExampleJsonSyncClass.synced.loadData();
+        ExampleJsonSyncClass instance = ExampleJsonSyncClass.synced.getObject();
+        instance.testInt = 10;
+        ExampleJsonSyncClass.synced.setFile(RESOURCE_DIRECTORY + "/SaveFileTest.json");
```

Comment:
will we have to set the file before every save? if so is there a way this could be incorporated into the save method to make it easier on robot code side?

Reply:
No, I just did that for testing, so it did not overwrite the file

Reply:
ok sounds good